gatsby-node.js

```diff
@@ -42,6 +42,107 @@ const getMessages = (path, language) => {
   }
 }
 
+/**
+ * Markdown isOutdated check
+ * Parse header ids in markdown file (both translated and english) and compare their info structure.
+ * If this structure is not the same, then the file isOutdated.
+ * If there is not english file, return true
+ * @param {string} path filepath for translated mdx file
+ * @returns boolean for if file is outdated or not
+ */
+const checkIsMdxOutdated = (path) => {
+  try {
+    const splitPath = path.split("/")
+    splitPath.splice(7, 2)
+    const englishPath = splitPath.join("/")
```

Comment:
I'm not sure this is right - don't you need to replace the lang (e.g. `ar`) with `en` in the path?

Reply:
I think as-is this only drops the `/content/translations/` from the path.

Reply:
I need to parse out without __dirname I think to get this right in hindsight. Will add that in.